src/main/java/bio/terra/common/opentelemetry/HttpMetricsAdvice.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package bio.terra.common.opentelemetry;
+
+import static java.util.Arrays.asList;
+
+import io.opentelemetry.api.metrics.DoubleHistogramBuilder;
+import io.opentelemetry.extension.incubator.metrics.ExtendedDoubleHistogramBuilder;
+import io.opentelemetry.instrumentation.api.semconv.http.internal.HttpAttributes;
+import io.opentelemetry.semconv.SemanticAttributes;
+import java.util.List;
+import java.util.stream.DoubleStream;
+
+/**
+ * copied from <a
+ * href="https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/release/v2.0.x/instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpMetricsAdvice.java">HttpMetricsAdvice</a>
+ * for the only purpose of overriding the DURATION_SECONDS_BUCKETS
+ */
+final class HttpMetricsAdvice {
+
+  // every 10ms up to 500ms, then every 100ms up to 2s, then every 1s up to 10s, then every 5s up to
+  // 1m, then every 10s up to 3m
+  static final List<Double> DURATION_SECONDS_BUCKETS =
+      DoubleStream.iterate(
+              0.01,
+              d -> d < (double) 60 * 3,
+              d -> {
+                if (d < .5) {
+                  return d + 0.01;
+                }
+                if (d < 2) {
+                  return d + 0.1;
+                }
+                if (d < 10) {
+                  return d + 1;
+                }
+                if (d < 60) {
+                  return d + 5;
+                }
+                return d + 10;
```

Comment:
Since this is Java 17, this could be a switch with guards.

Reply:
copilot wrote this code :)

Reply:
🤖 🚫

Reply:
I think guards is a preview feature of java 17 and we don't have that enabled

Reply:
ah, darn